### PR TITLE
Run tests for all HugetlbSizes

### PR DESCRIPTION
**Pull request by LK4D4**
This should fix CI failures

### CODE REVIEW

**Comment by LK4D4** [comment]
ping @mrunalp @crosbymichael

**Comment by mrunalp** [comment]
Awesome! :) LGTM

File: libcontainer/cgroups/fs/hugetlb_test.go [hunk 1]
```diff
@@ -32,38 +31,47 @@ func TestHugetlbSetHugetlb(t *testing.T) {
...
-		maxUsage: hugetlbMaxUsageContents,
-		failcnt:  hugetlbFailcnt,
-	})
+	for _, pageSize := range HugePageSizes {
+		helper.writeFileContents(map[string]string{
+			fmt.Sprintf(usage, pageSize):    hugetlbUsageContents,
+			fmt.Sprintf(maxUsage, pageSize): hugetlbMaxUsageContents,
+			fmt.Sprintf(failcnt, pageSize):  hugetlbFailcnt,
+		})
+	}
```
Comment (hqhq) [inline libcontainer/cgroups/fs/hugetlb_test.go]: I don't think test all HugetlbSizes for all test cases are necessary. The problem is `GetStats` will get all HugetlbSizes but we didn't create them in the fake cgroup directory. Can we just create all HugetlbSizes in this test case?

Comment (LK4D4) [inline libcontainer/cgroups/fs/hugetlb_test.go]: Why not just test for all?

Comment (hqhq) [inline libcontainer/cgroups/fs/hugetlb_test.go]: I think for testing of runc api, test for one hugetlb page size is good enough, they are exactly the same for other page sizes.

I'm thinking about fix like this:
```
diff --git a/libcontainer/cgroups/fs/hugetlb_test.go b/libcontainer/cgroups/fs/hugetlb_test.go
index 39c3ee6..5b89a47 100644
--- a/libcontainer/cgroups/fs/hugetlb_test.go
+++ b/libcontainer/cgroups/fs/hugetlb_test.go
@@ -59,11 +59,18 @@ func TestHugetlbSetHugetlb(t *testing.T) {
 func TestHugetlbStats(t *testing.T) {
        helper := NewCgroupTestUtil("hugetlb", t)
        defer helper.cleanup()
-       helper.writeFileContents(map[string]string{
-               usage:    hugetlbUsageContents,
-               maxUsage: hugetlbMaxUsageContents,
-               failcnt:  hugetlbFailcnt,
-       })
+       // hugetlb.GetStats() will get stats for all HugePageSizes,
+       // we need to create them here.
+       for _, pageSize := range HugePageSizes {
+               usage := strings.Join([]string{"hugetlb", pageSize, "usage_in_bytes"}, ".")
+               maxUsage := strings.Join([]string{"hugetlb", pageSize, "max_usage_in_bytes"}, ".")
+               failcnt := strings.Join([]string{"hugetlb", pageSize, "failcnt"}, ".")
+               helper.writeFileContents(map[string]string{
+                       usage:    hugetlbUsageContents,
+                       maxUsage: hugetlbMaxUsageContents,
+                       failcnt:  hugetlbFailcnt,
+               })
+       }

        hugetlb := &HugetlbGroup{}
        actualStats := *cgroups.NewStats()
```

Anyway, test for all page sizes does no harm, it's even good for functional test, so I'm OK with your fix, thanks. :)

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**